core/src/main/java/io/smallrye/openapi/api/OpenApiConfig.java

```diff
@@ -175,6 +175,10 @@ default Set<String> getScanExcludeProfiles() {
         return new HashSet<>();
     }
 
+    default Map<String, String> getScanResourceClasses() {
```

Comment:
It might be better to have this class return the class-name/path rather than the full configuration key. You'll want to strip out the prefix in your MavenConfig implementation of this and the PR will also need an implementation in `OpenApiConfigImpl` for the non-Maven case.

Reply:
I agree, will update it to return class name without prefix.
I'm not sure how should it be added to `OpenApiConfigImpl` since it gets properties from `org.eclipse.microprofile.config.Config` and I do not know how it gets populated.

Reply:
I think the method in `OpenApiConfigImpl` will be very similar to the `getSchemas` method.